app/models/tool.rb

```diff
@@ -37,95 +37,110 @@ class Tool < ApplicationRecord
     title: 'Name',
     type: 'single',
     show_in_table: true,
-    show_in_modal: true
+    show_in_modal: true,
+    show_in_csv: true
 
   table_attr :url,
     title: 'URL',
     type: 'single',
     show_in_table: true,
-    show_in_modal: true
+    show_in_modal: true,
+    show_in_csv: true
     
   table_attr :organisations,
     title: 'Lead organisation',
     type: 'multiple',
     show_in_table: false,
-    show_in_modal: true
+    show_in_modal: true,
+    show_in_csv: true
   
   table_attr :partners,
     title: 'Partners',
     type: 'multiple',
     show_in_table: false,
-    show_in_modal: false
+    show_in_modal: false,
+    show_in_csv: false
   
   table_attr :developers,
     title: 'Developers',
     type: 'multiple',
     show_in_table: false,
-    show_in_modal: false
+    show_in_modal: false,
+    show_in_csv: false
```

Comment:
Just a question;

If this has `false` for `show_in_table`, `show_in_modal` and `show_in_csv`, does it actually appear anywhere?

Reply:
No, it appears nowhere. I guess you are wondering why we have this table_attr in the first place ? I don't really know but I would assume that because it's a column of the csv (in the seeds), we need to have it as a table_attr.